### ISSUE TITLE
3rd game crash - 

### ISSUE THREAD

**Issue by Quietbeat** [ForgottenGlory/Living-Skyrim-3#51]
**LS Version**
3

**Describe the bug**
I had cleared out Fellglow Keep and as I tried entering the dungeon, the game became unresponsive so I had to kill the game manually. Similar to the other two crashes, there's nothing in the Overwrite folder. 

After restarting the game, this time I was able to enter the dungeons. The bad thing is that the crosshair was not displayed. When I went to Settings - Gameplay, I had to uncheck then re-check Crosshair.

**To Reproduce**
- Go to Fellglow Keep and clear it out 
- Try to enter the dungeon

**Expected behavior**
The game doesn't crash.

**Screenshots**
N/A

**Additional context**
[saves.zip](https://github.com/ForgottenGlory/Living-Skyrim-3/files/5922541/saves.zip)
![Crosshair missing](https://user-images.githubusercontent.com/57841662/106831955-ac094f80-6645-11eb-9e51-fb4da6dd3489.png)

[Crash.zip](https://github.com/ForgottenGlory/Living-Skyrim-3/files/5923418/Crash.zip)

**Comment by ForgottenGlory** [comment]
Check the contents of `Files Generated during playsessions launched with SKSE` and see if `NetScriptFramework\Crash` is located inside that for the crash log please.

**Comment by Quietbeat** [comment]
[Crash.zip](https://github.com/ForgottenGlory/Living-Skyrim-3/files/5928265/Crash.zip)

**Comment by daffodilian** [comment]
Are you positive you are playing the latest released version, and not the alpha build?

**Comment by ForgottenGlory** [comment]
From your crash logs, you have been playing on an Alpha build of the list which is not intended to be played. You will need to run the Wabbajack installer linked through the readme or server announcement channels to update the list. This will almost certainly require a new save for you. Unfortunately, I am not able to answer questions or fix issues from the Alpha build as we have moved on from that.

**Comment by Quietbeat** [comment]
I originally downloaded the build from the Read.Me and I still have that file. Can I verify if I have an Alpha or Beta build via :
- The file itself
- Info within Mod Manager 
- Any of the files that were installed

I downloaded the file again and it looks exactly like the one I originally downloaded.  
- How can I be sure that it's Beta and not Alpha?
- If I overwrite the existing installation, how would I know if I'm playing on the Alpha or Beta build?

**Comment by TwistedModding** [comment]
Download the beta wabbajack file from the discord and install.

**Comment by Quietbeat** [comment]
If you mean to click on the Installer link (image below) then this file looks like the other files.
![image](https://user-images.githubusercontent.com/57841662/106990658-86557680-6729-11eb-893f-6c30caddfe12.png)

You didn't address my questions about how to verify that it is Alpha or Beta.

**Comment by ForgottenGlory** [comment]
The one on Discord.

**Comment by ForgottenGlory** [comment]
Or, more specifically, the one in the readme and on Discord are the same. The only way to verify what version you're on is to reinstall the list using one of those.

**Comment by Quietbeat** [comment]
OK, I'll  reinstalling from Discord and afterwards, I'd like to be able to verify that I have the expected version. In Beta 2, can you include an unique identifier in a file?

**Comment by Quietbeat** [comment]
I just finished re-installing Beta 1 and I have a screenshot from Wabbajack (which I'm 99% sure was the same from when I originally installed the beta).  I cleared out the old crash files. I think I already ask this but, for Beta 2 can include a file that includes the version.

![Living Skyrim v3 Beta 1](https://user-images.githubusercontent.com/57841662/106992512-70e24b80-672d-11eb-812f-f9c4bce3563a.png)
